src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -248,6 +248,10 @@ public enum SlackDistributionFailureBehavior {
 
     public static final String VOLTAGE_TARGET_PRIORITIES_PARAM_NAME = "voltageTargetPriorities";
 
+    public static final String TRANSFORMER_VOLTAGE_CONTROL_STABLE_PARAM_NAME = "transformerVoltageControlStable";
+
+    public static final String TRANSFORMER_VOLTAGE_CONTROL_THT_LIMIT_PARAM_NAME = "transformerVoltageControlTHTLimit";
```

Comment:
`THT` is french... VHV ? HV ?

Reply:
thanks @jeandemanged . The parameter has be introduced in main with a different name by Anne. That shloud disappear when I merge with main